transform/mattermost-analytics/models/marts/release/dim_fix_versions.sql

```diff
@@ -1,42 +1,32 @@
 
-WITH unparsed_fix_versions AS (
-    -- Implement filtering on this layer as it's only used here.
-    SELECT
-        issue_id,
-        value:name::string as fix_version_name,
-        -- Break down different variations of target version
-        REGEXP_SUBSTR(fix_version_name, 'v\\d+\.\\d+') AS semver,
-        REGEXP_SUBSTR(fix_version_name, 'v(\\d+)', 1, 1, 'e', 1)::int AS version_major,
-        REGEXP_SUBSTR(fix_version_name, '\\.(\\d+)', 1, 1, 'e', 1)::int AS version_minor,
-        REGEXP_SUBSTR(fix_version_name, '\\.(\\d+)', 1, 2, 'e', 1)::int AS version_patch,
-        CASE
-            WHEN fix_version_name ILIKE '%mobile%' THEN 'Mobile'
-            WHEN fix_version_name ILIKE '%desktop%' THEN 'Desktop'
-            WHEN fix_version_name ILIKE '%playbooks%' THEN 'Playbooks'
-            WHEN fix_version_name ILIKE '%ir%' THEN 'IR'
-            WHEN fix_version_name ILIKE '%cloud%' THEN 'Cloud'
-            WHEN fix_version_name ILIKE '%apps%' THEN 'Apps'
-        END AS component,
-        TO_DATE(REGEXP_SUBSTR(fix_version_name, '\\d{2}/\\d{2}/\\d{2}'), 'mm/dd/yy') AS cloud_release_date
-    FROM
-        {{ ref('stg_mattermost_jira__issues') }},
-        LATERAL FLATTEN(INPUT => fix_versions)
-    WHERE
-        -- Keep only relevant fix versions - ones that contain a version in the form `v[major].[minor]`
-        REGEXP_LIKE(fix_version_name, '.*v\\d+\.\\d+.*', 'i')
-)
-SELECT
-    fv.issue_id,
-    fv.fix_version_name AS fix_version,
-    fv.semver,
-    fv.version_major,
-    fv.version_minor,
-    fv.version_patch,
-    fv.component,
-    fv.cloud_release_date,
-    rd.planned_release_date
-FROM
-    unparsed_fix_versions fv
-    -- Add planned release date by looking up dates ONLY for Cloud and On Prem releases)
-    LEFT JOIN {{ ref('stg_mattermost__version_release_dates') }} rd
-        ON fv.semver = rd.short_version AND (fv.component IS NULL OR fv.component = 'Cloud')
+-- Implement filtering on this layer as it's only used here.
+select
+    issue_id,
+    value:name::string as fix_version,
+    -- Break down different variations of target version
+    regexp_substr(fix_version, 'v\\d+\.\\d+') as semver,
+    regexp_substr(fix_version, 'v(\\d+)', 1, 1, 'e', 1)::int as version_major,
+    regexp_substr(fix_version, '\\.(\\d+)', 1, 1, 'e', 1)::int as version_minor,
+    regexp_substr(fix_version, '\\.(\\d+)', 1, 2, 'e', 1)::int as version_patch,
+    case
+        when fix_version ilike '%mobile%' then 'Mobile'
+        when fix_version ilike '%desktop%' then 'Desktop'
+        when fix_version ilike '%playbooks%' then 'Playbooks'
+        when fix_version ilike '%ir%' then 'IR'
+        when fix_version ilike '%cloud%' then 'Cloud'
+        when fix_version ilike '%apps%' then 'Apps'
+    end as component,
+    to_date(regexp_substr(fix_version, '\\d{2}/\\d{2}/\\d{2}'), 'mm/dd/yy') as cloud_release_date,
+    fix_version like any (
```

Comment:
would be nice to document/give an example to how a self-hosted `fix_version` would look like vs a cloud one.

Reply:
Added in docs.